src/Ydb.Sdk/src/Services/Topic/Reader/Reader.cs

```diff
@@ -0,0 +1,23 @@
+namespace Ydb.Sdk.Services.Topic.Reader;
+
+internal class Reader<TValue> : IReader<TValue>
+{
+    public Reader(Driver driver, ReaderConfig config, IDeserializer<TValue> deserializer)
+    {
+    }
+
+    internal Task Initialize()
+    {
+        throw new NotImplementedException();
+    }
+
+    public Task<TValue> ReadAsync()
+    {
+        throw new NotImplementedException();
+    }
+
+    public Task<Writer.Message<TValue>> ReadMessageAsync()
```

Comment:
Shouldn't the Message class be in some general package? This brings Writer dependency to Reader